.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -507,6 +507,8 @@ stages:
     timeoutInMinutes: 100
     strategy:
       matrix: $[stageDependencies.generate_variables.generate_variables_job.outputs['generate_variables_step.integration_tests_windows_matrix'] ]
+    variables:
+      USE_NATIVE_LOADER: true
```

Comment:
If we're going with "all caps" here, we should change the name of the existing variable in the pipeline configuration (in the Azure DevOps UI). Or remove it entirely.

Reply:
We need to change it, because on linux env var name are case-sensitive.
Not sure I have the rights to change/remove it.

Reply:
I just fixed it 🙂